src/store/auth/actions.js

```diff
@@ -0,0 +1,27 @@
+import {
+  LOGIN_REQUEST, LOGIN_FAILS, LOGIN_SUCCESS, LOGIN_SUBMIT
+} from './types'
+
+export const loginSubmit = ({ username, password }) => ({
+  type: LOGIN_SUBMIT,
+  username,
+  password
+})
+
+export const loginFails = (errorTxt, loading) => ({
+  type: LOGIN_FAILS,
+  errorTxt,
+  loading
+})
+
+export const loginSuccess = (sessionID, loading) => ({
+  type: LOGIN_SUCCESS,
+  sessionID,
+  loading
+})
+
+export const loginRequest = (loading, error) => ({
```

Comment:
you don't need the `loading` argument here. just set `loading` to `true` inside reducer

Reply:
also you don't need to pass `error` at all